### ISSUE TITLE
Rename to "autocomplete-python" and consolidate

### ISSUE THREAD

**Issue by mehcode** [tinloaf/autocomplete-plus-python-jedi#16]
For better discover-ability of this (awesome) package I recommend a rename to autocomplete-python and a consolidation (and removal of all but one) with the other autocomplete jedi plugins (I chose this package to post this issue too as it seems the most recent / working version).
- https://atom.io/packages/autocomplete-jedi
- https://atom.io/packages/autocomplete-plus-jedi

//cc @benogle

**Comment by tinloaf** [comment]
I don't think this is something we (me or @benogle) can or should do. Basically, that's other peoples packages, and just kicking them would be ... bad. Thanks for your support though. ;)

**Comment by mehcode** [comment]
I agree. However attempted communication with those packages and then (after some time period) removing them or flagging them someway should be done. If I search for "autocomplete python" right now I get a few different options and have to read through to figure it out. 

As a community we need to consolidate packages that provide the same functionality to better give a single package to the user.

**Comment by benogle** [comment]
It would be nice if everyone interested in this a python provider could work together on a single package cc @emhagman @fallenhitokiri

**Comment by tinloaf** [comment]
Yeah, absolutely. I wrote @fallenhitokiri a mail some time ago... perhaps we could get some coordination under way?

**Comment by fallenhitokiri** [comment]
@tinloaf just checked my mailbox again - I cannot find your mail. Did I ever answer it and forgot about it? Only thing I can find is your PR.

I have absolutely no problem removing my package from APM. I gave Atom a try and didn't find a maintained and working plugin, so I wrote one.
But since I like neither Atom nor the parts of the Atom / plugin community I encountered so far I am more than happy to see someone else shipping a working Python provider. I'll remove it this weekend.

Judging from this discussion it looks like the best course is to notify the users I am aware of to switch to this plugin?

**Comment by tinloaf** [comment]
@fallenhitokiri 30.3.2015 to timo@screamingatmyscreen.com ?
@benogle I had a look, and it seems there is no real possibility to merge two packages in APM, right? It's possible to rename them, though... could something be hacked with renaming?

**Comment by benogle** [comment]
cc @thedaniel, can we 'merge' package names?

**Comment by mehcode** [comment]
If we could make renaming work like it does in github than we could do a redirect from the current name to `autocomplete-python`. I'd also hide the "symlinked" names from search and inform the users that the name changed somehow (some warning or other).

**Comment by thedaniel** [comment]
Currently, when packages are renamed, there's a `redirect_from` attribute set on the package so the old name still works. I could fully delete (most records are soft-deleted) the package that should go away and manually set the `redirect_from` value on the one that stays.

**Comment by fallenhitokiri** [comment]
@tinloaf sorry - checked my inbox again and the spam filter, no mail from you. Not sure what went wrong.

**Comment by tinloaf** [comment]
Just seen that @sadovnychyi created yet another incarnation of this. See https://github.com/sadovnychyi/autocomplete-python/issues/4 .. we should definitely consolidate this somehow.
